spec/integration/kibana/tags.spec.yml

```diff
@@ -0,0 +1,44 @@
+##
+## Describes the specification for a routing rules yml file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: array
+  description: Tags automatically added to saved object assets.
+  items:
+     type: object
+     additionalProperties: false
+     properties:
+       text:
+         description: Tag name.
+         type: string
+       asset_types:
+         description: Asset types where this tag is going to be added.
+         type: array
+         items:
+           type: string
+           enum: # added the same types that are allowed under kibana folder
+             - dashboard
+             - visualization
+             - search
+             - map
+             - lens
+             - index_pattern
+             - security_rule
+             - csp_rule_template
+             - ml_module
+             - tag
```

Comment:
Umm, how are these tag assets different to the ones we are adding here?

Reply:
True, I just filled this enum list with all the types available in kibana folders. But that's true that `tag` would not be added other tags... I'll remove it from the enum list.

Reply:
About how these `tags` are different from the ones adding here. The tags added in this PR/issue are aimed to be shared tags between different dashboards/assets. So all of them will have the same tag (with the same id).